Test/linters/constructorMatch.dfy.expect

```diff
@@ -0,0 +1,10 @@
+constructorMatch.dfy(11,12): Warning: this branch is redundant 
+constructorMatch.dfy(17,12): Warning: this branch is redundant 
+constructorMatch.dfy(25,16): Warning: this branch is redundant 
+constructorMatch.dfy(9,17): Warning: Variable name 'ShadesOfGray' should begin with a lowercase letter
```

Comment:
There is no variable `ShadesOfGray` on line 9, column 17 of constructorMatch. Can you please figure out what's wrong, if any?